ls1intum/Artemis
# Show indents on the text exercise result

### Checklist

### Motivation and Context



### Description


### Steps for Testing


1. Log in to Artemis as an instructor
2. Navigate to Course Administration
3. Create a text exercise
4. Log in as a student
5. Participate in the exercise including tabs and space
6. Log in as an instructor
7. Assess the exercise
8. Log in as student
9. See that the tabs and spaces are show in the same way as they were created

### Test Coverage




### Screenshots


Screenshot before this PR:
![before](https://user-images.githubusercontent.com/20031592/121668838-5ee63000-cab4-11eb-99d2-23eab795733c.png)

Screenshot after this PR:
![now](https://user-images.githubusercontent.com/20031592/121668850-60aff380-cab4-11eb-9c0b-0b2699185ba5.png)
